src/rtcp/index.ts

```diff
@@ -0,0 +1,252 @@
+/*
+        0                   1                   2                   3
+        0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+       +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+header |V=2|P|    SC   |      PT       |             length            |
+       +=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
+ */
+
+/** @ignore */
+const RTCP_VERSION = 2;
+/** @ignore */
+const COMMON_HEADER_LENGTH = 4;
+
+export enum PacketType
+{
+	SR = 200,
+	RR = 201,
+	SDES = 202,
+	BYE = 203,
+	APP = 204,
+	RTPFB = 205,
+	PSFB = 206,
+	XR = 207
+}
+
+/**
+ * Inspect the given buffer and return a boolean indicating whether it could be
+ * a valid RTCP packet or not.
+ *
+ * ```ts
+ * if (isRtcp(buffer)) {
+ *   console.log('it seems a valid RTCP packet');
+ * }
+ * ```
+ */
```

Comment:
```suggestion
/**
 * ```ts
 * import { isRtcp } from 'rtp.js';
 * ```
 *
 * Inspect the given buffer and return a boolean indicating whether it could be
 * a valid RTCP packet or not.
 *
 * ```ts
 * if (isRtcp(buffer)) {
 *   console.log('it seems a valid RTCP packet');
 * }
 * ```
 */
```